buildSrc/build.gradle.kts

```diff
@@ -98,6 +102,7 @@ dependencies {
     compile("org.asciidoctor:asciidoctor-gradle-plugin:1.5.6")
     compile("com.github.javaparser:javaparser-core:2.4.0")
     compile("com.google.code.gson:gson:2.7")
+    compile("com.gradle:build-scan-plugin:1.12.1")
```

Comment:
Does this mean that we now cannot change the version of the build scan plugin which is applied to the `gradle/gradle` build itself from an init script?

Reply:
See comment above. @ldaley needs to answer.